pkg/connector/tickets.go

```diff
@@ -342,27 +317,41 @@ func (d *Connector) GetTicketSchema(ctx context.Context, schemaID string) (*v2.T
 		return schema, nil, nil
 	}
 
-	project, err := d.jiraClient.GetProject(ctx, schemaID)
+	// Format is projectKey:issueID
+	schemaIDParts := strings.Split(schemaID, ":")
+	if len(schemaIDParts) != 2 {
+		return nil, nil, errors.New("schema ID format should be projectKey:issueTypeID")
```

Comment:
That go type should do all this. in a go way that returns an error on marshal/unmarshal